pydruid/client.py

```diff
@@ -564,7 +565,7 @@ def _post(self, query):
                     pass
 
             raise IOError(
-                "{0} \n Druid Error: {1} \n Query is: {2}".format(
+                "{0}\n Druid Error: {1}\n Query is: {2}".format(
```

Comment:
Is this necessary?

Reply:
It generated trailing whitespace in the tests and flake8 didn't like it. I can leave it as it was and add a `noqa` it the test so it doesn't complain.

Reply:
Ok, I was confused. It wasn't `flake8` but the `trailing-whitespace` hook from `pre-commit-hooks`. I think that hook doesn't have a way to skip a particular line :thinking: I'm not sure how it made it here since the `upstream` project also [asks to use it for contributing](https://github.com/druid-io/pydruid/#contributing). Since the hook automatically removes the trailing whitespace in the tests, it makes them fail if I don't change how the strings are created here.

Reply:
I'll create a PR to fix this in the `upstream` project as well.